### DISCUSSION TITLE
HAJK på Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by matself** [hajkmap/Hajk#1473] [Q&A] [answered]
Jag har gett upp på Docker och gick på en vanilla NodeJS-installation istället.
Så vitt jag kan bedöma snurrar allt, backend, client och admin på maskinen, men jag har stött på ett CORS-problem. Admin ger respons, men misslyckas hämta lagerlistan. Kartan dyker genast efter spinnern och säger nätverksfel.
Dagen har gått åt till att läsa förslag, fråga ChatGPT och testa testa testa. Inget hjälper. Någon måste ha satt upp HAJK på Ubuntu?
Här är Debug-konsolen från en standard node-nginx-setup. Jag bjuder på webbadressen som ni ser, om ni vill prova själva.
```
The Cross-Origin-Opener-Policy header has been ignored, because the URL's origin was untrustworthy. It was defined either in the final response or a redirect. Please deliver the response using the HTTPS protocol. You can also use the 'localhost' origin instead. See https://www.w3.org/TR/powerful-features/#potentially-trustworthy-origin and https://html.spec.whatwg.org/#the-cross-origin-opener-policy-header.
geowebb.se/:1 The page requested an origin-keyed agent cluster using the Origin-Agent-Cluster header, but could not be origin-keyed since the origin 'http://geowebb.se' had previously been placed in a site-keyed agent cluster. Update your headers to uniformly request origin-keying for all pages on the origin.
FetchWrapper.js:205 
 GET http://localhost:3002/api/v2/config/map_1 net::ERR_CONNECTION_REFUSED
FetchWrapper.js:205 
 GET http://localhost:3002/api/v2/config/map_1 net::ERR_CONNECTION_REFUSED
index.js:49 TypeError: Failed to fetch
    at u (FetchWrapper.js:205:10)
    at g (index.js:122:24)
    at async Promise.all (/index 0)
geowebb.se/:1 Uncaught (in promise) Error: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
﻿
```

**Comment by jesade-vbg** [comment, marked as answer]
Hej!

Ser ut som du kör på geowebb.se

Prova att ta bort 
http://localhost:3002
från appconfig.json

Url:en börjar då med /api/v2 osv.

**Comment by jesade-vbg** [comment]
Verifierade lite snabbt och din map_1 ligger här
http://geowebb.se/api/v2/config/map_1
Så mitt föregående svar bör vara korrekt.

**Comment by matself** [comment]
STORT Tack! Det löste problemet. Jag undrar vad jag gjorde fel?
I vilket fall, samma ändring (många) i /static/admin/config.json löste motsvarande problem med Admin-gränssnittet också.
Sedan är det så att de tjänster som kommer med default från Länsstyrelsen som börjar med https://ext-geodata.lansstyrelsen.se/arcgis/services/WMS... inte finns längre. De har lagt om det mesta till ArcGIS REST-tjänster.
Nu kan jag äntligen komma vidare med ett projekt som har hamnat i bakvattnet flera år.
